### PR TITLE
Add .jabbarc

### PR DESCRIPTION
**Pull request by Sebkasanzew**
To specify supported Java versions for this project, when using jabba, the Java version manager.

### CODE REVIEW

**Comment by reicheltp** [comment]
Gradle specifies which JAVA version is required. Thus, I see no need to add yet another file with redundant information.

**Comment by Sebkasanzew** [comment]
> Gradle specifies which JAVA version is required. Thus, I see no need to add yet another file with redundant information.

This file is helpful for developers with multiple Java projects on one machine, who use Jabba (like nvm) to switch beween java versions. Gradle does not download and switches the version Java by itself.

File: .jabbarc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# https://github.com/shyiko/jabba
+jdk: adopt@1.8.0-222
```
Comment (reicheltp) [inline .jabbarc]: Can this be wildcarded to any JDK matching 1.8.* ? 

Does this enforce adopt JDK? Is it still working if a user has installed OpenJDK or Oracle?

Comment (Sebkasanzew) [reply .jabbarc]: I updated the required version to be a wildcard, but `adopt@1.8.0-222` is somehow sorted above `@1.14` ... but other OpenJDK and Oracle versions work with this wilrcard.

**Pull request merged**